frontend/src/scenes/insights/BreakdownFilter/TaxonomicBreakdownFilter.tsx

```diff
@@ -103,7 +99,7 @@ export function BreakdownFilter({ filters, setFilters, buttonType }: TaxonomicBr
 
         if (changedBreakdownType) {
             let newFilters: Partial<FilterType>
-            if (multiPropertyBreakdownIsEnabled) {
+            if (useMultiBreakdown) {
```

Comment:
I think there are changes in master that will clash here

Reply:
ah cool I'll rebase

Reply:
💥